code/__DEFINES/mobs.dm

```diff
@@ -468,3 +468,15 @@
 
 //Saves a proc call, life is suffering. If who has no targets_from var, we assume it's just who
 #define GET_TARGETS_FROM(who) (who.targets_from ? who.get_targets_from() : who)
+
+///Ensures mobs that have recently stopped crawling can be easily hit for a little longer. See below.
+#define CRAWLING_PENALTY (0.3 SECONDS + world.tick_lag*2)
```

Comment:
world.tick_lag * 2???

Reply:
Maybe I should just remove it since the penalty is never going to be 0 or less.